src/utils/components/NetworkIcons/PlugCircleCheckIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+
+type PlugCircleCheckIconProps = {
+  className?: string;
+  fill?: string;
+  height?: number | string;
+  width?: number | string;
+};
+
+const PlugCircleCheckIcon: FC<PlugCircleCheckIconProps> = ({ className, fill, height, width }) => (
```

Comment:
I'm sure about the icon - it's good enough but on/off or connected/disconnected icons are also OK for me https://www.patternfly.org/design-foundations/icons/#all-icons